src/App.js

```diff
@@ -1,13 +1,22 @@
 import './App.css';
 import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Header from './components/Header';
+import Home from './components/Home';
+import Quote from './components/Quote';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] OPTIONAL 
You have **disabled linter check** on your code. Please enable linter check on your code to ensure a cleaner syntax.